examples/serving/serving.tensorflow.json

```diff
@@ -0,0 +1,21 @@
+{
+    "jobName": "tensorflow-serving",
+    "image": "paiexample/pai.example.tensorflow-serving",
+    "taskRoles": [
+      {
+        "name": "serving",
+        "taskNumber": 1,
+        "cpuNumber": 4,
+        "memoryMB": 8192,
+        "gpuNumber": 1,
+        "portList": [
+          {
+            "label": "model_server",
+            "beginAt": 0,
+            "portNumber": 1
+          }
+        ],
+        "command": "bazel-bin/tensorflow_serving/example/mnist_saved_model /tmp/mnist_model && tensorflow_model_server --port=$PAI_CONTAINER_HOST_model_server_PORT_LIST --model_name=mnist --model_base_path=/tmp/mnist_model"
```

Comment:
>bazel-bin/tensorflow_serving/example/mnist_saved_model /tmp/mnist_model && tensorflow_model_server --port=$PAI_CONTAINER_HOST_model_server_PORT_LIST --model_name=mnist --model_base_path=/tmp/mnist_model [](start = 20, length = 202)

For Service, please warp a while loop for this, so that it will always retry even on success. Please check Launcher user manual, 

Launcher will not restart the succeeded Task (i.e. the process started by EntryPoint ends with exit code 0) in any RetryPolicy. So, if you want to always restart Service on the same machine irrespective of its exit code, you need to warp the original EntryPoint by another script, such as:

 while true; do
     # call the original EntryPoint
 done

Reply:
updated